### ISSUE TITLE
Enhancment:- On hover it should mention what 40/40 means  ...(credits left)

### ISSUE THREAD

**Issue by Aarize29** [ai-graphic/Graphic-so#1]
Issue:- In this side navigation bar , there is a part where it shows 40/40 , but it does not have informantion about what 40/40 really means

Enhancement:- We can add a hover, upon hover we will get a mesaage denoting that it is CREDITS LEFT
![image](https://github.com/user-attachments/assets/cda1e844-0092-4af8-bce5-ba94a89dc33b)

**Comment by seshubonam** [comment]
yeah spot on. can have a "credits-remaining" title and hover view with more description as you suggested.

**Comment by Aarize29** [comment]
can I contribute to this issue?

**Comment by seshubonam** [comment]
yes sure send a PR

**Comment by Aarize29** [comment]
I am facing little issue on setting up the project locally, could help me to setup it on my machine. Just write the steps so that I can follow it.

**Comment by rohit-sama** [comment]
Hey Aarize29, thanks for reaching out. Could you please provide more details about the specific issues you're encountering after following our README?

**Comment by rohit-sama** [comment]
Hey Aarize29, thanks for reaching out. Could you please provide more details about the specific issues you're encountering after following our README?

**Comment by Aarize29** [comment]
yaah sure,  so I have setup the codebase locally and changed the  example env file to env file, but when  i am   doing npx prisma db push , it just stays there for a long time.


like this
![image](https://github.com/user-attachments/assets/2fef4ef3-f204-4a0f-9c33-62133ae76c61)

**Comment by rohit-sama** [comment]
Hey Aarize29, this could be a database issue, we are using neon for database, so try using the same db. also could be network issues as well so try changing your network if switching to neon still doesn't work.

here is the link for [neonDb](https://neon.tech/).

**Comment by Aarize29** [comment]
Hey, I have added the hover thing... and created a pull request, can you please verify it or check it once !!!
